test/integration/bootstrap/bootstrap_test.go

```diff
@@ -15,20 +15,114 @@
+		tft.WithTFDir("../../../1-bootstrap"),
 	)
 
 	bootstrap.DefineVerify(func(assert *assert.Assertions) {
 		bootstrap.DefaultVerify(assert)
 
+		projectID := bootstrap.GetStringOutput("project_id")
+		gcloudArgsBucket := gcloud.WithCommonArgs([]string{"--project", projectID, "--json"})
+
+		bucketReposBuild := []string{
+			"mt-build",
+			"af-build",
+			"fs-build",
+		}
+		for _, bucket := range bucketReposBuild {
+			urlBucket := fmt.Sprintf("https://www.googleapis.com/storage/v1/b/bkt-%s-%s", projectID, bucket)
+			opBucket := gcloud.Run(t, fmt.Sprintf("storage ls --buckets gs://bkt-%s-%s", projectID, bucket), gcloudArgsBucket).Array()
+			assert.Equal(urlBucket, opBucket[0].Get("metadata.selfLink").String(), fmt.Sprintf("The bucket name should be %s.", urlBucket))
+			assert.True(opBucket[0].Exists(), "Bucket %s should exist.", urlBucket)
+		}
+
+		bucketReposLogs := []string{
+			"mt-logs",
+			"af-logs",
+			"fs-logs",
+		}
+		for _, bucket := range bucketReposLogs {
+			urlBucket := fmt.Sprintf("https://www.googleapis.com/storage/v1/b/bkt-%s-%s", projectID, bucket)
+			opBucket := gcloud.Run(t, fmt.Sprintf("storage ls --buckets gs://bkt-%s-%s", projectID, bucket), gcloudArgsBucket).Array()
+			assert.Equal(urlBucket, opBucket[0].Get("metadata.selfLink").String(), fmt.Sprintf("The bucket name should be %s.", urlBucket))
+			assert.True(opBucket[0].Exists(), "Bucket %s should exist.", urlBucket)
+		}
+
+		bucketRepoState := []string{
+			"tf-state",
+		}
+		for _, bucket := range bucketRepoState {
+			urlBucket := fmt.Sprintf("https://www.googleapis.com/storage/v1/b/bkt-%s-%s", projectID, bucket)
+			opBucket := gcloud.Run(t, fmt.Sprintf("storage ls --buckets gs://bkt-%s-%s", projectID, bucket), gcloudArgsBucket).Array()
+			assert.Equal(urlBucket, opBucket[0].Get("metadata.selfLink").String(), fmt.Sprintf("The bucket name should be %s.", urlBucket))
+			assert.True(opBucket[0].Exists(), "Bucket %s should exist.", urlBucket)
+		}
+
+		repos := []string{
+			"eab-applicationfactory",
+			"eab-fleetscope",
+			"eab-multitenant",
+		}
+		for _, repo := range repos {
+			url := fmt.Sprintf("https://source.developers.google.com/p/%s/r/%s", projectID, repo)
+			repoOP := gcloud.Runf(t, "source repos describe %s --project %s", repo, projectID)
+			repoSa := fmt.Sprintf("serviceAccount:tf-cb-%s@%s.iam.gserviceaccount.com", repo, projectID)
+			repoIamOpts := gcloud.WithCommonArgs([]string{"--flatten", "bindings", "--filter", "bindings.role:roles/viewer", "--format", "json"})
+			repoIamPolicyOp := gcloud.Run(t, fmt.Sprintf("source repos get-iam-policy %s", repo), repoIamOpts).Array()[0]
+			listMembers := utils.GetResultStrSlice(repoIamPolicyOp.Get("bindings.members").Array())
+			assert.Contains(listMembers, repoSa, fmt.Sprintf("Service Account %s should have role roles/viewer on repo %s", repoSa, repo))
+			assert.Equal(url, repoOP.Get("url").String(), "source repo %s should have url %s", repo, url)
+		}
+
+		triggerRepos := []string{
```

Comment:
Can use the other variable `repos` (line 74)
Maybe just change the name (or not)